grafana/k6-docs
# fix: Search form fix (reload on Enter key)

This pull request adds small fix to prevent default behaviour (reload on Enter key) in SearchForm:


**Checklist**:


**Steps to test**
1. Open the [page](https://mdr-ci.staging.k6.io/docs/refs/pull/890/merge);
2. Confirm that everything looks and works as expected;


Closes: #868
